.dockerignore

```diff
@@ -9,8 +9,11 @@
 **/__pycache__
 **/venv
 **/var
-**/resources/**/generated
-**/resources/portal/static/*
+
+resources/**/generated
+resources/portal/static/*
+!resources/portal/static/img/*
```

Comment:
Oh why I thought the build pipeline should generate the images?

Reply:
The images inside img is static.

Reply:
The real problem is we use the same directory to contain generated assets and static assets. But that is another issue.